include/ajax.tasks.php

```diff
@@ -424,7 +422,7 @@ function massProcess($action, $w=null)  {
 
             // Assume success
             if ($i==$count) {
-                $msg = sprintf(__('Successfully %s %s.'),
+                $msg = sprintf(__('Successfully %1$s %2$s.' /* Tokens are <actioned> <x selected task(s)> */),
```

Comment:
Ok, so everything stays as it is or do you plan to make changes in the background so the comment can be moved outside the function?